README.md

```diff
@@ -1,21 +1,56 @@
-# perl-Data-Resample
+# perl-Data-Decimate
 
-Data::Resample
+A module that allows you to decimate a data feed
```

Comment:
Can you explain what that means? Remember this module is on CPAN. Any random Perl coder who comes accross it needs to understand what the module does.